v2/emailpassword/common-customizations/sessions/with-jwt/jwt-verification.mdx

```diff
@@ -80,13 +80,22 @@ Refer to this [Github gist](https://gist.github.com/rishabhpoddar/8c26ed237add1a
   - This function takes care of caching public keys in memory + auto refetching if the public keys have changed (which happens automatically every 24 hours with supertokens). This will not cause any user logouts, and is just a security feature.
 - `main.go`: This is an example of how to verify a JWT using the golang JWT verification lib along with our helper function to get the JWKs keys. If you are using header based auth, you can fetch the JWT from the `Authorization Bearer` header, otherwise for cookie based auth, you can fetch it from the `sAccessToken` cookie.
 
+</TabItem>
+<TabItem value="java">
+
+Refer to this [Github gist](https://gist.github.com/ConzorKingKong/47e2723ede54079149f5f7c11e44c164) for a code reference of how use the Java `nimbus-jose-jwt` lib to do session verification. The gist contains three files:
```

Comment:
In the gist, why does `JOSEException e` yield a 500 and not 401?

Reply:
Changed to a 401